client/packages/common/src/ui/components/inputs/Select/Select.tsx

```diff
@@ -17,33 +19,69 @@ const defaultRenderOption = (option: Option) => (
 );
 
 export const Select: FC<SelectProps> = React.forwardRef(
-  ({ options, renderOption, sx, InputProps, ...props }, ref) => (
-    <TextField
-      ref={ref}
-      sx={{
-        '& .MuiInput-underline:before': { borderBottomWidth: 0 },
-        '& .MuiInput-input': { color: theme => theme.palette.gray.dark },
-        ...sx,
-      }}
-      select
-      variant="standard"
-      size="small"
-      InputProps={{
-        disableUnderline: true,
-        ...InputProps,
-        sx: {
-          backgroundColor: theme =>
-            props.disabled
-              ? theme.palette.background.toolbar
-              : theme.palette.background.menu,
-          borderRadius: '8px',
-          padding: '4px 8px',
-          ...InputProps?.sx,
-        },
-      }}
-      {...props}
-    >
-      {options.map(renderOption || defaultRenderOption)}
-    </TextField>
-  )
+  (
+    {
+      options,
+      renderOption,
+      sx,
+      InputProps,
+      clearable = false,
```

Comment:
Clearable `false` by default as I thought it might be best to not mess with existing Select elements. Happy to change it to `true` though if we want to see the "clear" icon on existing elements?

Reply:
think thats cool, IMHO it should be configured in the UIComponent based on the if the json field is required or if the user overwrote the option...